components/Home/HotDeals/HotDeals.js

```diff
@@ -1,44 +1,63 @@
 import styled from "styled-components";
+import Link from "next/link";
 
-import { ItemCard, Button, Divider } from "../../../styles/UIKit";
-import { HotDealsData } from "../../../config/HomeConfig";
-
-const HotDeals = () => (
-  <HotDealsContainer>
-    <HotDealsTitleContainer>
-      <HotDealsTitle>{HotDealsData.title}</HotDealsTitle>
-      <ViewAllLink>{HotDealsData.linkText} </ViewAllLink>
-    </HotDealsTitleContainer>
-    <Divider />
-    <HotDealsContent>
-      <HotDealsOfferContainer>
-        <HotDealsOfferTitle>{HotDealsData.OfferText}</HotDealsOfferTitle>
-        <HotDealsCounterContainer>
-          <HotDealsCounterSquare>
-            <SquareCounter>100</SquareCounter>
-            <SquareTime>DAYS</SquareTime>
-          </HotDealsCounterSquare>
-          <HotDealsCounterSquare>
-            <SquareCounter>23</SquareCounter>
-            <SquareTime>HOURS</SquareTime>
-          </HotDealsCounterSquare>
-          <HotDealsCounterSquare>
-            <SquareCounter>36</SquareCounter>
-            <SquareTime>MINS</SquareTime>
-          </HotDealsCounterSquare>
-          <HotDealsCounterSquare>
-            <SquareCounter>45</SquareCounter>
-            <SquareTime>SECS</SquareTime>
-          </HotDealsCounterSquare>
-        </HotDealsCounterContainer>
-        <ShopNowButton inverted> {HotDealsData.buttonText}</ShopNowButton>
-      </HotDealsOfferContainer>
-      {HotDealsData.items.map((item) => (
-        <ItemCard key={item.id} itemName={item.name} itemPrice={item.price} />
-      ))}
-    </HotDealsContent>
-  </HotDealsContainer>
-);
+import { Button, Divider } from "../../../styles/UIKit";
+import { HeaderData, HotDealsData } from "../../../config/HomeConfig";
+import { useEffect, useState } from "react";
```

Comment:
Library import and file imports should be separate, and move react imports to the top of the file

Reply:
This has been taken care .